src/action.test.js

```diff
@@ -137,6 +137,36 @@ test("should return the number of linked issues and their repos using loose matc
   ]);
 });
 
+test.each([["pull_request"], ["pull_request_target"]])(
+  "should succeed when PR is labeled with 'no-issue'",
```

Comment:
can you please interpolate the event name in the test name so we have tests with different names, instead of duplicates?